render/egl.c

```diff
@@ -712,23 +714,39 @@ static char *get_render_name(const char *name) {
 }
 
 int wlr_egl_dup_drm_fd(struct wlr_egl *egl) {
-	if (egl->device == EGL_NO_DEVICE_EXT || !egl->exts.EXT_device_drm) {
+	if (egl->device == EGL_NO_DEVICE_EXT || (!egl->exts.EXT_device_drm &&
+			!egl->exts.EXT_device_drm_render_node)) {
```

Comment:
Wont this check fail if the user doesnt have the ext?

Reply:
If the user doesn't have `EXT_device_drm_render_node`, but has `EXT_device_drm`, then the `if` is not taken. The `if` is only taken if the user is missing both exts.

Reply:
I should learn to read throughly the conditions for next time, sorry

Reply:
No worries!